src/utils/menu.js

```diff
@@ -1,52 +1,85 @@
 const menu = [
   {
-    name: 'Apps',
+    name: '1hive Products',
     sublinks: [
       {
-        name: 'Honey Faucet',
-        link: 'https://faucet.1hive.org/',
-        description: 'Register with BrightId to claim Honey'
+        name: 'Honeypot',
+        link: 'https://1hive.org',
+        description: 'DAO Governance'// - Create and stake on proposals'
```

Comment:
As above, it's not really governance (most of the time)

Reply:
See above, I can change to The DAO proposals

Reply:
i would just give a name something like "DAO interface" that is what really is

Reply:
Can we shy away from interface and give it a more specific noun of what they are taking action, everything pretty much is an interface.

Reply:
mmmm the thing is that are not the only things that you can do in that interface, you can also execute the issuance, you can have your profile, see each other activities on the DAO, etc

Reply:
Well I can see we have some strong stances with this. I politely disagree with needing to leave it general. You can argue that it is more than just an interface as well. But to get the ball rolling on the main objective of adding a link to the farm page I will change it. Thank you for your responsiveness and honest feedback.

Reply:
So based on feedback it sounds like this is acceptable?

**New Proposal:**
Card - Honeypot - The DAO to create and stake on distribution proposals.
Menu - Honeypot - The DAO Interface

_For reference - Current State:
Card - 1Hive DAO Interface - Create and stake on Honey distribution proposals
Menu - 1Hive DAO Interface - Create Proposals and Stake Honey_